emperor/pandas.py

```diff
@@ -0,0 +1,116 @@
+r"""
+Pandas Interface for Emperor
+============================
+
+This module provides a simple interface to visualize a Pandas DataFrame using
+the Emperor.
+
+.. currentmodule:: emperor.pandas
+
+Functions
+---------
+.. autosummary::
+    :toctree: generated/
+
+    scatterplot
+"""
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, emperor development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.md, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+import numpy as np
+import pandas as pd
+
+from emperor.core import Emperor
+from skbio import OrdinationResults
+
+
+def scatterplot(df, x=None, y=None, z=None, remote=True):
+    """Create an Emperor scatter plot from a Pandas DataFrame
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Pandas DataFrame with the data to display
```

Comment:
Alright, I'm a little confused here.

Does this dataframe contain everything?  Metadata and PC coordinates?

I wouldn't think this would be desirable - our OTU tables and metadata are often decoupled, and this would put quite a 
bit of burden on the user to get this sorted out.

In the [canvas biplot code](https://github.com/mortonjt/canvas/blob/master/canvas/biplot.py) we keep all of the metadata and PC coordinates separate.  And I think this greatly enhances useability.